views/editor.dt

```diff
@@ -19,20 +19,20 @@ block content
 		div(style="padding-left: 0px; padding-right: 0px")
 			div#command-box.text-right
 				button.btn.btn-primary(ng-click="run()")
-					i.fa.fa-play(aria-hidden="true")
 					span Run
-				button.btn.btn-primary(ng-click="format()")
 					i.fa.fa-play(aria-hidden="true")
+				button.btn.btn-primary(ng-click="format()")
```

Comment:
I think the format button should use the `btn-default` style. It has the same importance as `Reset`, `Shorten`, etc., IMO.